content/en-us/education/battle-royale-series/coding-the-game-loop.md

```diff
@@ -182,23 +182,23 @@ With a **repeat loop**, the code in the loop will run at least once. Unlike a wh
 
 At this point, if you're not spawning as intended, try one of the following below.
 
-- `wait()` should be inside the repeat loop. Without the wait, the script will run too many times in a second, overloading Roblox Studio and causing an error.
+- `task.wait()` should be inside the repeat loop. Without the wait, the script will run too many times in a second, overloading Roblox Studio and causing an error.
 - In the Game Settings module, the variable `intermissionDuration` should be greater than 1. If lower, the script can repeat too often, causing slow down issues.
 
 ### Ending the Intermission
 
 Once there are enough players, have them wait a short transition time. Then, send them into the match by calling the `prepareGame()` function in MatchManager. Remember, that function just prints a line, but you'll add more code later.
 
-1. At the end of the repeat loop, add a print statement saying the intermission is over to test your code. Then, follow it with a `wait()` using GameSetting's `transitionTime` variable.
+1. At the end of the repeat loop, add a print statement saying the intermission is over to test your code. Then, follow it with a `task.wait()` using GameSetting's `transitionTime` variable.
```

Comment:
Adding `Libary.` makes the code automatically link to the [reference page](https://create.roblox.com/docs/reference/engine/libraries/task#wait) (see [STYLE.md](https://github.com/Roblox/creator-docs/blob/main/STYLE.md#links)):

```suggestion
1. At the end of the repeat loop, add a print statement saying the intermission is over to test your code. Then, follow it with a `Library.task.wait()` using GameSetting's `transitionTime` variable.
```